app/client/src/ce/sagas/PageSagas.tsx

```
@@ -536,15 +536,15 @@ export function* savePageSaga(action: ReduxAction<{ isRetry?: boolean }>) {
         );
 
         if (actions && actions.length) {
-          yield put(setActionsToExecuteOnPageLoad(actions));
+          yield put(setActionsRunBehavior(actions));
         }
 
         const jsActions = actionUpdates.filter((d) =>
           d.hasOwnProperty("collectionId"),
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)